### ISSUE TITLE
menu styling

### ISSUE THREAD

**Issue by MortenHofft** [gbif/hp-beaty-biodiversity-museum#42]
You've chosen to go to mobile layout on all devices. 
But I find that it makes it more difficult to use the menu since the items are all the way to the left. Would it make sense to add a bit of style to make it right aligned?

Could be as simple as this in the main.scss file?

```css
.navbar-menu {
    float: right;
    padding: 1rem 2rem;
}
```

<img width="408" alt="Screenshot 2023-11-24 at 22 09 30" src="https://github.com/gbif/hp-beaty-biodiversity-museum/assets/6759267/3801be15-a5db-467a-b0b5-482bad8090d5">

**Comment by mpitblado** [comment]
Playing around with it, I think I like the smaller menu even on mobile. I'm going to push this to staging so I can see how it feels on my phone.  Small modifications so that the menu does not change other cards in the interface to shift from center when opened:

```scss
  .navbar-menu {
    position:absolute;
    right: 0;
    padding: 1rem 2rem;
  }
```

 Also realizing I have been spoiled by tailwindcss for too long!

**Comment by mpitblado** [comment]
Works quite well on mobile because the buttons are right there for the thumb. Thanks for the suggestion! Closed by #43

**Comment by MortenHofft** [comment]
If you like tailwindcss and doing html css, then it is perfectly possible to build your own layouts and still use the specimen search etc. Also within Jekyll. Just stop using the predefined layouts.
